src/Umbraco.Cms.Integrations.SEO.Semrush/Api/Management/Controllers/GetCurrentContentPropertiesController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Integrations.SEO.Semrush.Configuration;
+using Umbraco.Cms.Integrations.SEO.Semrush.Models.Dtos;
+using Umbraco.Cms.Integrations.SEO.Semrush.Services;
+
+namespace Umbraco.Cms.Integrations.SEO.Semrush.Api.Management.Controllers
+{
+    [ApiVersion("1.0")]
+    [ApiExplorerSettings(GroupName = Constants.ManagementApi.SemrushGroupName)]
+    public class GetCurrentContentPropertiesController : SemrushControllerBase
```

Comment:
This is the alternate approach that I use in order to get properties and their values of current content and display it on the UI. Since CMS team have proposed another approach to get the properties in client, I will remove this in the next PR update.